template/design/api.go

```diff
@@ -14,7 +14,7 @@ var _ = API("{{camel Name "-" | title}}", func() {
 	Produces("application/json")
 
 	Security(auth.JWT(), func() {
-		Scope(auth.ScopeAPIAccess)
+		Scope("api:access")
```

Comment:
set to a const?

Reply:
scopes will be more specific for services, "api:access" is just an example required scope for a freshly generated one